src/phlorest/commands/check.py

```diff
@@ -1,6 +1,7 @@
 """
 Checks datasets for compliance
 """
+import sys
```

Comment:
Made it back in, but isn't used.

Reply:
ahh, I'd used sys.exit at some point. I'll remove this in main now.